overlord/overlord.go

```diff
@@ -254,6 +260,11 @@ func (o *Overlord) SetRestartHandler(handleRestart func(t state.RestartType)) {
 func (o *Overlord) Loop() {
 	o.ensureTimerSetup()
 	o.loopTomb.Go(func() error {
+		if o.udevMon != nil {
```

Comment:
Same, move this to ensure of the new manager.

Reply:
Done (although Ensure is atm a bit artificial as you'll see).